src/events/language.js

```diff
@@ -0,0 +1,27 @@
+// Event: name
+const Users = require('../server/users.service');
+const consts = require('../server/consts');
+
+// Initialize event listener
+module.exports = function(socket) {
+    socket.on('language', (language) => {
+
+        // TODO emit error
+        if (!consts.SUPPORTED_LANGUAGES.includes(language)) {
+            console.error(`Language ${language} not supported`);
```

Comment:
I assume the errors will sent back to the client will be implemented later

Reply:
Yes, this part is not ready yet